src/main/java/Parser.java

```diff
@@ -26,14 +26,38 @@ public void getCommand(String input,Storage store, Ui ui){
                 store.setDoneAt(index-1);
                 ui.printLine();
                 ui.indentPrint("Nice! I've marked this task as done: ");
-                ui.indentPrint("  "+ui.taskToString(store.getTask(index-1)));
+                ui.indentPrint("  "+store.getTask(index-1).toString());
+                ui.printLine();
+                break;
+            case "todo":
+                store.addToDoToTemp(others);
+                ui.printLine();
+                ui.indentPrint("Got it. I've added this task: ");
+                ui.indentPrint("  "+store.getTask(store.getSize()-1).toString());
+                ui.indentPrint( "Now you have "+store.getSize()+" tasks in the list.");
+                ui.printLine();
+                break;
+            case "deadline":
+                store.addDeadlineToTemp(others);
+                ui.printLine();
+                ui.indentPrint("Got it. I've added this task: ");
+                ui.indentPrint("  "+store.getTask(store.getSize()-1).toString());
+                ui.indentPrint( "Now you have "+store.getSize()+" tasks in the list.");
+                ui.printLine();
+                break;
+            case "event":
+                store.addEventToTemp(others);
+                ui.printLine();
+                ui.indentPrint("Got it. I've added this task: ");
+                ui.indentPrint("  "+store.getTask(store.getSize()-1).toString());
+                ui.indentPrint( "Now you have "+store.getSize()+" tasks in the list.");
                 ui.printLine();
                 break;
```

Comment:
I felt these codes are highly repeated with each other, but don't know how to make it nicer.

Any suggestions?

Reply:
I would consolidate the steps (print the message, print the task)  and write a new function to handle it

Reply:
Sounds great! I will try to work in this direction.